RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -1114,6 +1116,42 @@ class DeepTauId : public deep_tau::DeepTauBase {
                           const CellGrid& grid,
                           bool is_inner) {
     tensorflow::Tensor& convTensor = *convTensor_.at(is_inner);
+
+    eGammaTensor_[is_inner] = std::make_unique<tensorflow::Tensor>(
+        tensorflow::DT_FLOAT,
+        tensorflow::TensorShape{
+            (long long int)grid.num_valid_cells(), 1, 1, dnn_inputs_2017_v2::EgammaBlockInputs::NumberOfInputs});
```

Comment:
this approach increases the size of the tensors in memory.
My very rough calculations add up to about 250 kB extra if the grids are fully filled.
This seems acceptable.
If this is correct, no action is needed.

Reply:
That's roughly what I get as well. 
In practice, the grids should be quite sparse (<10% filled based on https://indico.cern.ch/event/829640/contributions/3473229/attachments/1869221/3097000/2019-06-26_DeepTau_ID_computational_performance.pdf).